### ISSUE TITLE
Reexporting/resubmitting a submission sets its submission state to Exported even if it's still exporting

### ISSUE THREAD

**Issue by aaclan-ebi** [ebi-ait/dcp-ingest-central#220]
happened for a dcp3 project https://contribute.data.humancellatlas.org/submissions/detail?uuid=c6216c95-8b02-44c3-bffb-0a454ceeea7a&project=c1810dbc-16d2-45c3-b45e-3e675f88d87b

and might be also happening for new projects that will be reexported...

**Comment by clairerye** [comment]
I don't think we touched this ticket this sprint? Do we have plans for it? @aaclan-ebi or @jacobwindsor do either of you know?

**Comment by yusra-haider** [comment]
I remember that @jacobwindsor worked on this ticket last to last sprint: https://github.com/ebi-ait/ingest-state-tracking/pull/7
